src/styles/index.scss

```diff
@@ -1,3 +1,61 @@
-body {
+@import 'vars';
+
+.page__body {
   margin: 0;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  box-sizing: border-box;
+}
+
+.calendar {
+  display: flex;
+  flex-flow: row wrap;
+  gap: $day-gap;
+  width: $calendar-width;
+  padding: 10px;
+
+  &__day {
+    width: $day-width;
+    height: $day-width;
+    background-color: $calendar-color;
+    color: black;
+    border: $day-border;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+
+    &:hover {
+      cursor: pointer;
+      background-color: $pointer-calendar-color;
+      transform: translateY(-20px);
+      transition-duration: 500ms;
+    }
+
+    @for $numbers from 1 through 31 {
+      &:nth-child(#{$numbers})::before {
+        content: '#{$numbers}';
+      }
+    }
+  }
+
+  @each $day, $number in $days {
+    &--start-day-#{$day} {
+      .calendar__day:first-of-type {
+        margin-left: $number * ($day-width + $day-gap);
+      }
+    }
+  }
+
+  @for $numbers from 28 through 31 {
+    &--month-length-#{$numbers} {
+      .calendar__day:nth-child(n + #{$numbers + 1}) {
+        display: none;
+      }
+    }
+  }
```

Comment:
The `@for` directive is used here, but it seems that the `$numbers` variable is not defined in this file. Make sure that the variable is imported from the 'vars.scss' file or declared in this file.